QuizAppNew/View/QAview.swift

```diff
@@ -9,8 +9,12 @@ import SwiftUI
 
 //MARK: Question Answer view
 struct QAview: View {
+    @State var isSubmitted = false
     @Environment(\.presentationMode) var present
     var body: some View {
+        if isSubmitted {
```

Comment:
You can't use **if else** direct in **var body: some View {** without SwiftUI parent Building layouts such as HStack, VStack, ZStack etch